orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/kubernetes/KubernetesContainerFinder.groovy

```diff
@@ -63,6 +63,15 @@ class KubernetesContainerFinder {
 
     containers.forEach { container ->
       if (container.imageDescription.fromContext) {
+	//  If there are parallel Find images stages , then deploymentDetails would be fetched from it. 
```

Comment:
@ajordens   

Hi , after debugging this one , Seems like  Multiple threads are competing with each other to write to a global context  and yes for every Find stage you  have the amiDetails for each stage context  which is written to global context based on race condition winner of two threads (or multiple) .   so without the support  of different variable , how we can populate the global context where second thread has already overwritten the context. 
Please can you  guide me for the above 


cc @robfletcher  @duftler